lib/src/views/chatroom_page.dart

```diff
@@ -53,22 +54,25 @@ class _ChatRoomPageState extends State<ChatRoomPage> {
   User? user;
 
   int currentTime = DateTime.now().millisecondsSinceEpoch;
-  Map<String, List<Media>> conversationAttachmentsMeta = <String, List<Media>>{};
+  Map<String, List<Media>> conversationAttachmentsMeta =
+      <String, List<Media>>{};
```

Comment:
Please set the line break to 80 characters if not set already, a lot of unnecessary changes are there

Reply:
Line break is already set to 80 char.